mod-source-record-manager-server/src/main/java/org/folio/services/progress/OptionalCodec.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.services.progress;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.eventbus.MessageCodec;
+
+import java.util.Optional;
+
+/**
+ * Codec that won't cause any serialization/deserialization
+ */
+public class OptionalCodec implements MessageCodec<Optional<?>, Optional<?>> {
```

Comment:
could you say please why we need OptionalCodec codec? As far as understand a BatchableJobExecutionProgress object always is passed for event bus producer.

Reply:
Good catch! A previous implementation used this codec to notify the event bus producer that the batch action was completed. But it slowed data import jobs so i turned it to fire and forget. I have removed this codec since it is no longer needed.